src/django/api/reports/average_list_count_per_contributor.sql

```diff
@@ -0,0 +1,14 @@
+select
```

Comment:
I don't have a real preference but I feel like I've usually seen SQL written with keywords capitalized. However, lowercase is clear enough.

Now that we have raw SQL files in the repository (and the possibility of adding more in the future, or changing these in some way), would it make sense to add a SQL linter to run as part of the `lint` script -- assuming one exists?

Reply:
It's faster to write things when you never worry about the shift key, but I checked the Postgres docs and the SQL generated by the Django ORM and they both use all-caps keywords. I'll go ahead and upcase them for consistency.

I did some Googling but did not find a command line SQL linter.